File: app/src/main/java/com/example/mynewapplication/javapackage/ContentDetailActivity.java
package com.example.mynewapplication.javapackage;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.widget.ImageButton;

import com.example.mynewapplication.R;

public class ContentDetailActivity extends AppCompatActivity {

    private WebView webView;
    private WebSettings webSettings;
    private ImageButton ib_back;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_content_detail);

        webView = findViewById(R.id.wv_detail);
        ib_back = findViewById(R.id.ib_back_detail);

        ib_back.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

        webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true);
        webView.getSettings().setAppCacheEnabled(true);
        webView.getSettings().setDatabaseEnabled(true);
        webView.getSettings().setDomStorageEnabled(true);
        webView.getSettings().setJavaScriptEnabled(true);
        webView.getSettings().setBuiltInZoomControls(false);
        webSettings.setAllowFileAccess(true);
        webSettings.setAppCacheEnabled(true);
        webSettings.getUseWideViewPort();
        webView.setWebViewClient(new WebViewClient());
        webView.loadUrl(getIntent().getStringExtra("href"));
    }
}

File: app/src/main/java/com/example/mynewapplication/ContentDetailActivity.kt
package com.example.mynewapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageButton

class ContentDetailActivity : AppCompatActivity() {

    private lateinit var imageButton: ImageButton

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_content_detail)

        imageButton = findViewById(R.id.ib_back)
        imageButton.setOnClickListener {
//            startActivity(Intent(this, ContentListActivity::class.java))
            finish()
        }
    }
}